lib/workers/repository/update/branch/execute-post-upgrade-commands.ts

```diff
@@ -61,6 +67,40 @@ export async function postUpgradeCommandsExecutor(
         }
       }
 
+      let dataFilePath = '';
+      if (dataFileTemplate) {
+        const dataFileContent = compile(
+          dataFileTemplate,
+          mergeChildConfig(config, upgrade),
+        );
+        logger.debug(
+          { dataFileTemplate },
+          'Processed post-upgrade commands data file template.',
+        );
+
+        const tmpDir = os.tmpdir();
```

Comment:
Can I suggest we use `fs.mkdtemp` instead (with `import fs from 'fs-extra'`), maybe best abstracted inside `lib/util/fs`?

Reply:
we probably should use our private cache dir instead, which is already cleaned after each repo.

https://github.com/renovatebot/renovate/blob/3b6b00bf66e9134c0cfdc96a7d8793fbfdafb0ee/lib/modules/manager/helmv3/common.ts#L53-L59